src/App.tsx

```diff
@@ -1,8 +1,33 @@
-import React from 'react';
-
-export const App: React.FC = () => (
-  <div className="App">
-    <p>Nothing was pressed yet</p>
-    <p>The last pressed key is [Enter]</p>
-  </div>
-);
+import { Component } from 'react';
+
+type State = {
+  pressedKey: string,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
```

Comment:
it's better to save here exactly key and render additional text directly in render method